corehq/sql_accessors/sql_templates/get_case_attachment_by_identifier.sql

```diff
@@ -1,8 +0,0 @@
-DROP FUNCTION IF EXISTS get_case_attachment_by_identifier(TEXT, TEXT);
```

Comment:
You should copy the `DROP` statements into a new migration. You'll also need to remove any references to these files in migrations.

Reply:
Also these should probably be in a separate, followup PR in case there are any dependencies on these functions